### ISSUE TITLE
Chest sliders are not updated after drag & drop

### ISSUE THREAD

**Issue by gizosate** [im-mi/KoikatuDragAndDrop#3]
Hi, 
I noticed, chest parameters from previous girl are kept when I drag & drop another card to the window. But if I do it second time, chest is updated properly. 
Issue doesn't occur when I load card via game interface, only after drag & drop. Also this might not happen all the time, but when it's started to happen, I can reproduce it constantly.

Thanks

**Comment by im-mi** [comment]
I haven't been able to reproduce this. Are you by chance running any other plugins? I wonder if something like the push-up bra plugin could be causing it.

Thanks for the bug report.

**Comment by gizosate** [comment]
You are right, these two files of  were causing this glitch:
BepInEx\IPA
KK_Kiyase.dll
KK_Kiyase.ini
Moved them, and now can not reproduce it.
Related to Bra_Push-Up_Mod provided with the HF patch 1.3 I think.

I hope either you or developer of Bra_Push-Up_Mod could fix it.
Thank you!

**Comment by im-mi** [comment]
I did a little research and learned that this happens due to a limitation of how the push-up bra / kiyase mod detects when a character is loaded.

The kiyase mod keeps track of the original slider values at all times. That way it can modify them to suit the current outfit and then revert back to the originals when necessary. To get said original slider values, it watches for the character load button to get pressed and then uses that moment to copy slider values from the newly loaded character. However, the drag and drop mod completely bypasses said button, so the kiyase mod doesn't notice when a character is loaded via drag and drop. As a result, it ends up using slider values from the wrong character.

That said, I'm considering this an oversight in kiyase rather than drag and drop.
